src/api/singa.i

```diff
@@ -25,10 +25,5 @@
 %include "config.i"
 %include "core_tensor.i"
 %include "core_device.i"
-%include "model_layer.i"
-%include "model_optimizer.i"
-%include "model_loss.i"
-%include "model_metric.i"
 %include "model_operation.i"
-%include "io_snapshot.i"
-%include "dist_communicator.i"
```

Comment:
the dist_communicator.i is the interface between c and python, without `include "dist_communicator.i"` the commuicator cannot be found in python code

```
======================================================================
ERROR: test_dist (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_dist
Traceback (most recent call last):
  File "/usr/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/usr/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/root/dcsysh/singa/test/python/test_dist.py", line 27, in <module>
    sgd = opt.DistOpt(sgd)
  File "/root/dcsysh/singa/build/python/singa/opt.py", line 362, in __init__
    self.communicator = singa.Communicator(buffSize)
AttributeError: module 'singa.singa_wrap' has no attribute 'Communicator'

```